### PR TITLE
[Feature: #164136755] Return User Profile Details when user is authenticated

### PR DESCRIPTION
**Pull request by habibaudu**
#### What does this PR do?

- Return User Profile Details when user is authenticated

#### Description of Task to be completed?

- [x] Make an include statement for `profile`, `following` `followers` `notification` in database query  for users .
- [x] Returns result of query to users

#### How should this be manually tested?

- On terminal 
- Run `npm run dev`
- Fire up postman 
- POST => `localhost:4000/users/login`
- POST => `localhost:4000/users`

#### What are the relevant pivotal tracker stories?

[#164136755](https://www.pivotaltracker.com/story/show/164136755)
#### Screenshots (if appropriate)
screen shot 2019-02-21 at 2 22 12 pm

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -167,10 +167,74 @@ class Users {
       username: user.username
     };
 
+
+    const getUserProfile = await User.findOne(
```
Comment (oreoluwade) [inline controllers/user.js]: This block of code is exactly the same with the one below. You will have to extract that into a function and call it in `login` and `register` user.

**Review by oreoluwade — CHANGES_REQUESTED** [review]
Dropped a comment for you.

File: controllers/user.js (hunk 1 above)
Comment (habibaudu) [reply controllers/user.js]: ok

Comment (habibaudu) [reply controllers/user.js]: I have made changes

**Review by oreoluwade — APPROVED** [review]

**Pull request merged**